### PR TITLE
Thomas folbrecht

### PR DESCRIPTION
**Pull request by tfolbrecht**

### CODE REVIEW

File: assignments/array-methods.js [hunk 1]
```diff
@@ -56,22 +56,40 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
...
 console.log(largeShirts);
 
 // ==== Challenge 4: Use .reduce() ====
 // The donations need to be tallied up and reported for tax purposes. Add up all the donations into a ticketPriceTotal array and log the result
-let ticketPriceTotal = [];
-console.log(ticketPriceTotal);
+// let ticketPriceTotal = [];
+// sum all prices
+let ticketPriceTotal = 
+runners.reduce((total, runner) => total + runner.donation, 0)
```
Comment (gsamaniego41) [inline assignments/array-methods.js:91]: Nice job converting this to a one liner. It looks cleaner. :)

File: assignments/closure.js [hunk 2]
```diff
@@ -1,20 +1,28 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+const phrase = "Lambda school is cool";
+function returnPhrase () {
+  return phrase;
```
Comment (gsamaniego41) [inline assignments/closure.js:5]: Not quite an example of a closure. The idea is that inner functions have access to variables declared in their outer functions
```
function returnPhrase() {
  const phrase = "Lambda school is cool";

  return function() {
    return phrase
  }
}
```

Here are two ways of invoking it:
```
returnPhrase.()()
```
```
const newPhraseFunc = returnPhrase()
newPhraseFunc()
```

**Review by gsamaniego41 — COMMENTED** [review]
MVP met! Nice job.

File: assignments/closure.js (hunk 2 above)
Comment (tfolbrecht) [reply assignments/closure.js:5]: On my todo!

**Pull request merged**